android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/uimodel/SurveyQuestionUiModel.kt

```diff
@@ -0,0 +1,12 @@
+package co.nimblehq.kaylabruce.kmmic.android.presentation.uimodel
+
+data class SurveyQuestionUiModel(
+    val imageUrl: String,
+    val questionNumber: Int,
+    val totalQuestion: Int,
+    val questionTitle: String,
+    val isLastQuestion: Boolean,
+) {
+    val questionIndex: String
+        get() =  "${questionNumber}/${totalQuestion}"
```

Comment:
(Optional) We could remove the bracelets here
```suggestion
        get() =  "$questionNumber/$totalQuestion"
```